### PR TITLE
Update post-ti-indicator.md

### PR DESCRIPTION
**Pull request by msbemba**
Updated the action setting "GenerateAlert" is an item that must be set to "TRUE" when creating an action with "Audit".

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/6717

File: microsoft-365/security/defender-endpoint/post-ti-indicator.md [hunk 1]
```diff
@@ -75,7 +75,7 @@ Parameter|Type|Description
 :---|:---|:---
 indicatorValue|String|Identity of the [Indicator](ti-indicator.md) entity. **Required**
 indicatorType|Enum|Type of the indicator. Possible values are: "FileSha1", "FileMd5", "CertificateThumbprint", "FileSha256", "IpAddress", "DomainName" and "Url". **Required**
-action|Enum|The action that will be taken if the indicator will be discovered in the organization. Possible values are: "Alert", "Warn", "Block", "Audit, "BlockAndRemediate", "AlertAndBlock", and "Allowed". **Required**
+action|Enum|The action that will be taken if the indicator will be discovered in the organization. Possible values are: "Alert", "Warn", "Block", "Audit, "BlockAndRemediate", "AlertAndBlock", and "Allowed". **Required**. "GenerateAlert" is an item that must be set to "TRUE" when creating an action with "Audit".
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/post-ti-indicator.md]:
```suggestion
action|Enum|The action that will be taken if the indicator will be discovered in the organization. Possible values are: "Alert", "Warn", "Block", "Audit, "BlockAndRemediate", "AlertAndBlock", and "Allowed". **Required**. The "GenerateAlert" parameter must be set to "TRUE" when creating an action with "Audit".
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @msbemba, kindly commit the suggested changes if you agree with it. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@mjcaparas Please merge this PR. Thanks

**Pull request merged**